src/java/org/apache/cassandra/schema/ColumnMetadata.java

```diff
@@ -73,6 +74,11 @@ public boolean isPrimaryKeyKind()
 
     }
 
```

Comment:
Changes made to `ColumnMetadata` in DB-3460 are on top of a pervious update for DB-3434 that introduced some dropped column related code.
So the changes here for DB-3460  include the pre-requisite changes needed from DB-3434.